crane/network.go

```diff
@@ -0,0 +1,33 @@
+package crane
+
+import (
+	"fmt"
+	"os"
+)
+
+type Network interface {
+	Name() string
+	Create()
+	Exists() bool
+}
+
+type network struct {
+	RawName string
+}
+
+func (n *network) Name() string {
```

Comment:
should the prefix flag/env variable also apply as namespace for networks/volumes? I'd be tempted to say yes